packages/writing-style/vale-styles/commercetools/Acronyms.yml

```diff
@@ -8,6 +8,7 @@ second: '(?:\b[A-Z][a-z]+ )+\(([A-Z]{3,5})\)'
 # ... with the exception of these:
 exceptions:
   - API
+  - AWS
```

Comment:
Is that ok for you?

Reply:
Yes!  It's derived from the Google style so it's no surprise AWS is not an allowed word :-)